judgels-frontends/raphael/src/components/MathjaxWrapper/MathjaxWrapper.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+declare global {
+  interface Window {
+    MathJax: any;
+  }
+}
+
+export class MathjaxWrapper extends React.Component {
+  componentDidMount() {
+    this.insertScript('MathJax-config', '/var/conf/mathjax-config.js');
```

Comment:
I don't think this would work. `/var` is a Docker volume during deployment (https://github.com/ia-toki/judgels/blob/master/deployment/ansible/roles/raphael-deploy/tasks/main.yml#L31), so the content of the directory will come from user.

Why not just inside `/public`?

Reply:
Yes, this is actually inside `/public` folder. I used `process.env.PUBLIC_URL` in the `insertScript` method (https://github.com/lungsin/judgels/blob/33e65dbe9fc08067f2306abc77ab1324b7b09fb2/judgels-frontends/raphael/src/components/MathjaxWrapper/MathjaxWrapper.tsx#L23). 

I will change the code so that it is more intuitive.